packages/instantsearch.js/src/lib/insights/listener.tsx

```diff
@@ -58,7 +58,7 @@ const insightsListener = (BaseComponent: any) => {
         if (targetWithEvent) {
           const payload = parseInsightsEvent(targetWithEvent);
 
-          payload.forEach((single) => props.sendEvent!(single));
+          payload.forEach((single) => props.sendEventToInsights!(single));
```

Comment:
Would it work if we just rely on the fixed `sendEvent` prop that correctly references the method from the connector?

Reply:
Let me check, it should indeed work as well!

Reply:
yep that worked. Implemented in 3aa05880e